Sandbox/Gameplay/src/SandboxGame.cpp

```diff
@@ -40,6 +40,11 @@ namespace Sandbox
 		Input::BindKeyPress<&Sandbox::Game::ToggleRotation>(KeyNames::FL_KEY_T, *this);
 		Input::BindKeyPress<&Sandbox::Game::OnToggleMoveLights>(KeyNames::FL_KEY_SPACE, *this);
 
+		// Switch between window modes 
+		Input::BindKeyPress<&Sandbox::Game::SetWindowFullscreen>(KeyNames::FL_KEY_2, *this);
```

Comment:
I would prefer to see this as a ImGUI dropdown, or at the very least display some text to the user "press 1 to go fullscreen, etc"